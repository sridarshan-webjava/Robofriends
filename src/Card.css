.flow-content > * + * {
  margin-top: 1.5rem;
}

.card {
  display: inline-block;
  width: 18rem;
  margin: 1rem;
  padding: 1rem 0.5rem;
  background: #9ef2e2;
  box-shadow: 0 0 5px 0 rgba(0, 0, 0, 0.25);
  transition: transform 0.25s ease-in-out;
  text-align: center;
  border-radius: 0.75rem;
}

.card:hover {
  transform: scale(1.1);
}

.card img {
  width: 200px;
  height: 200px;
}

.card h2 {
  font-size: 1.35rem;
  font-weight: 700;
  color: #5d6c6b;
}

.card p {
  font-weight: 400;
}
