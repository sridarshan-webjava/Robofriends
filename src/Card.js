import React from "react";
import "./Card.css";

const Card = ({ name, email, id }) => {
  return (
    <div className="card flow-content">
      <img alt="robot" src={`https://robohash.org/${id}?size=200x200`} />
      <h2>{name}</h2>
      <p>{email}</p>
    </div>
  );
};

export default Card;
