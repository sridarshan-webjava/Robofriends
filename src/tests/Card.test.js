import React from "react";
import renderer from "react-test-renderer";
import Card from "../Card";

it("renders correctly", () => {
  const tree = renderer
    .create(<Card name="Sample" email="sample@xyz.com" id="123" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
