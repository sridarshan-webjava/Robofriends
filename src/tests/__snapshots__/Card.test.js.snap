// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="card flow-content"
>
  <img
    alt="robot"
    src="https://robohash.org/123?size=200x200"
  />
  <h2>
    Sample
  </h2>
  <p>
    sample@xyz.com
  </p>
</div>
`;
